tests/unsorted/floor_divmod.sby

```diff
@@ -0,0 +1,44 @@
+[options]
+mode bmc
+
```

Comment:
please add the option "depth 1" here, it speeds up the test and makes no difference otherwise for something purely combinational